internal/cmd/profile/login_command.go

```diff
@@ -303,3 +302,32 @@ func persistAccessCredentials(creds *session.StoredCredentials) error {
 		Credentials: creds,
 	})
 }
+
+func serveOnOpenPort(handler func(w http.ResponseWriter, r *http.Request)) (*http.Server, int, error) {
+	addr, err := net.ResolveTCPAddr("tcp", ":0")
```

Comment:
Tested this on windows to see if it actually works: 
![image](https://user-images.githubusercontent.com/40318863/218412147-b295d0f4-09a2-4227-8714-907489c2cb32.png)